supportedBackends/google/pipelines/batch/src/main/scala/cromwell/backend/google/pipelines/batch/ProjectLabels.scala

```diff
@@ -0,0 +1,3 @@
+package cromwell.backend.google.pipelines.batch
```

Comment:
This package has already many classes, I'd suggest creating `cromwell.backend.google.pipelines.batch.models` instead

Reply:
Thanks!  Adjusted.